app/_data/tables/version_errors_konnect.yml

```diff
@@ -928,6 +929,208 @@ messages:
+      updates to fail and potentially getting the data plane into an out-of-sync
+      state.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.1.0.0` or above.
+    DocumentationURL: |
+      [LDAP Authentication Advanced plugin](/hub/kong-inc/ldap-auth-advanced/)
+  - ID: D190
+    Severity: error
+    Description: |
+      For the `openid-connect` plugin, one or more of the following `config` fields
+      are set in a reference format: `client_id`, `client_secret`, `session_secret`,
+      `session_redis_username`, `session_redis_password` but Konnect only supports
+      Vaults starting from Kong Gateway version 3.1.0.0. This plugin instance will
+      cause the Kong Gateway to fail payload validation, causing payload updates to
+      fail and potentially getting the data plane into an out-of-sync state.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.1.0.0` or above.
+    DocumentationURL: |
+      [OpenID Connect plugin](/hub/kong-inc/openid-connect/)
+  - ID: D191
+    Severity: error
+    Description: |
+      For the `request-transformer-advanced` plugin, one or more of the following
+      `config` fields are set in a reference format: `rename.body`, `rename.headers`,
+      `rename.querystring`, `replace.body`, `replace.headers`, `replace.querystring`,
+      `add.body`, `add.headers`, `add.querystring`, `append.body`, `append.headers`,
+      `append.querystring` but Konnect only supports Vaults starting from Kong Gateway
+      version 3.1.0.0. This plugin instance will cause the Kong Gateway to fail
+      payload validation, causing payload updates to fail and potentially getting the
+      data plane into an out-of-sync state.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.1.0.0` or above.
+    DocumentationURL: |
+      [Request Transformer Advanced plugin](/hub/kong-inc/request-transformer-advanced/)
+  - ID: D192
+    Severity: error
+    Description: |
+      For the `graphql-rate-limiting-advanced` plugin, instances have been configured
+      with `strategy=redis` and the `cluster_addresses` field set. This is not
+      supported in Kong Gateway versions < 3.4.1.0. For data plane nodes with version
+      < 3.2.0.0, the plugin has been completely removed.
+    Resolution: |
+      Unset the `cluster_addresses` field for the plugin, or upgrade Kong Gateway to a
+      version >= 3.4.1.0.
+    DocumentationURL: |
+      [Graphql Rate Limiting Advanced plugin](/hub/kong-inc/graphql-rate-limiting-advanced/)
+  - ID: D193
+    Severity: error
+    Description: |
+      For the `graphql-rate-limiting-advanced` plugin, instances have been configured
+      with `strategy=redis` and the `cluster_addresses` field set. This is not
+      supported in Kong Gateway versions < 3.4.1.0. For data plane nodes with version
+      >= 3.2.0.0 and < 3.4.1.0, the plugin configuration has been changed to
```

Comment:
```suggestion
      supported in Kong Gateway versions < 3.4.1.0. For data plane nodes with 
      version >= 3.2.0.0 and < 3.4.1.0, the plugin configuration has been changed to
```
The markdown converter is reading the `>` as a blockquote tag:
<img width="627" alt="Screenshot 2023-11-06 at 1 18 40 PM" src="https://github.com/Kong/docs.konghq.com/assets/54370747/3c9aa459-3878-4556-b1a1-9e5f0d5260e9">